packages/coinstac-ui/app/render/components/results/result.jsx

```diff
@@ -213,7 +224,36 @@ class Result extends Component {
               Open Local Results
             </Button>
           </div>
-
+          <div className={classes.resultButton}>
+            {run.shouldUploadAssets && (
+              <Button
+                disabled={!run.assetsUploaded || downloading || filesExist}
+                variant="contained"
+                color="primary"
+                style={{ marginLeft: 10 }}
+                onClick={async () => {
+                  const authToken = JSON.parse(localStorage.getItem(API_TOKEN_KEY)).token;
+                  const clientId = user.id;
+                  const { apiServer } = window.config;
+                  const apiServerUrl = `${apiServer.protocol}//${apiServer.hostname}${apiServer.port ? `:${apiServer.port}` : ''}`;
+                  this.setState({ downloading: true });
+                  try {
+                    const result = await ipcRenderer.invoke('download-run-assets', {
+                      runId: run.id, authToken, clientId, apiServerUrl,
+                    });
+                    console.log(result);
+                    this.setState({ downloading: false });
+                    await this.doFilesExist(run.id);
+                  } catch (e) {
+                    console.log(e);
+                    this.setState({ downloading: false });
```

Comment:
I'm guessing the top log is for debug, but the bottom should probably be wired to an error toast/notification